src/frontend/less/index.less

```diff
@@ -1,3 +1,13 @@
 body {
-  background: white
+  background: white;
+}
+
+.login-form {
+  margin: 50px auto;
+  text-align: center;
+  width: 400px;
+}
+
+.login-form > * {
```

Comment:
Is this matching every child element of `.login-form`? That seems pretty broad?

Reply:
**Direct** children only, so it's not as broad. If you think about it, what I want is to all elements in the form to be centralized with that much space between them. And I can always override individually since this is a very broad match, compared to a more specific tag match, for example.